.ci/pipeline.yml

```diff
@@ -18,7 +18,7 @@ shared:
       export GOPATH=${PWD}/go
       export PATH="$GOPATH/bin:$PATH"
       cd code
-      make lint
+      make IS_CI=1 lint
```

Comment:
It doesn't have an impact, as it's just for verbosity; but for the note, as this is an anchor both jobs referencing it should have been modified for proper testing ;)

Not only https://github.com/cycloidio/terracognita/blob/66a08fd6845165acae754e2b5c8ef8571c1b760e/.ci/pipeline.yml#L183 but also ̀test-master` https://github.com/cycloidio/terracognita/blob/66a08fd6845165acae754e2b5c8ef8571c1b760e/.ci/pipeline.yml#L217